### PR TITLE
Fix several vulnerabilities in the NPM dependencies

### PR DESCRIPTION
**Pull request by russelarms**
Things to consider before you submit the PR:

* [x] Has `CHANGELOG.md` been updated?
* [x] Are tests passing locally?
* [x] Are the files formatted correctly?
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

- Updated **jest** to fix **acorn** vulnerability
- Pinned **minimist** version inside `appcenter-link-scripts` to the patched version https://www.npmjs.com/advisories/1179
- Updated **metro** version to fix **mem** vulnerability https://npmjs.com/advisories/1084
- Updated **react-native-screens** to use a stable build version

Tested the change with DemoApp and TestApp.

## Related PRs or issues

[AB#77963](https://msmobilecenter.visualstudio.com/Mobile-Center/_workitems/edit/77963)

### CODE REVIEW

File: appcenter-link-scripts/package.json [hunk 1]
```diff
@@ -23,7 +23,8 @@
     "mkdirp": "0.5.3",
     "plist": "3.0.1",
     "which": "1.2.11",
-    "xcode": "2.0.0"
+    "xcode": "2.0.0",
+    "minimist": "1.2.5"
```
Comment (guperrot) [inline appcenter-link-scripts/package.json]: This one is shipped, let's update Changelog.

File: appcenter/package.json [hunk 2]
```diff
@@ -31,7 +31,7 @@
   },
   "devDependencies": {
     "babel-eslint": "7.2.3",
-    "babel-jest": "21.0.2",
+    "babel-jest": "25.1.0",
```
Comment (guperrot) [inline appcenter/package.json]: Same here for changelog.

**Comment by dhei** [comment]
Looks good except addressing the changelog comments.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,11 @@
 # App Center SDK for React Native Change Log
 
+## Version 3.1.0 (Under development)
```
Comment (dhei) [inline CHANGELOG.md]: No new feature or behavior change, only bugfix.

```suggestion
## Version 3.0.1 (Under development)
```

**Pull request merged**